Source/FullScreenPokemon.ts

```diff
@@ -8540,7 +8542,11 @@ module FullScreenPokemon {
         }
 
         /**
+         * Saves the position of a certain character.
          * 
+         * @param FSP
+         * @param character   The current character object.
```

Comment:
Upper-case 'C' for character. Also, it's not 'current' so a different explanation (say, `an in-game Character` or something like that) should be used.